src/App.jsx

```diff
@@ -14,33 +15,118 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState([...goodsFromServer]);
+  const [sortState, setSortState] = useState({
+    reversed: false,
+    lengthSorted: false,
+    alphabetSorted: false,
+  });
+  const [isReset, setIsReset] = useState(false);
```

Comment:
```suggestion
  const [sortParams, setSortParams] = useState({
    isReversed: false,
    sortType: '',
  });
```
Why do you need so many states? It could be simplified to one state with sort parameters